linera-execution/src/committee.rs

```diff
@@ -99,8 +99,9 @@ impl Epoch {
 impl Committee {
     pub fn new(validators: BTreeMap<ValidatorName, ValidatorState>) -> Self {
         let total_votes = validators.values().fold(0, |sum, state| sum + state.votes);
-        // Let N = 3f + 1 + k (0 <= k < 3).
-        // * (2 N + 3) / 3 = 2f + 1 + (2k + 2)/3 = 2f + 1 + k = N - f
+        // Let N = 3f + 1 + k (0 <= k <= 2).
+        // * k / 3 = 0, (2 - k) /3 = 0 (0 <= 2 - k <= 2)
```

Comment:
It's true that the comment was not amazing. How about this version?
```
Let N = 3f + 1 + k such that 0 <= k <= 2. (Notably ⌊k / 3⌋ = 0 and ⌊(2 - k) / 3⌋ = 0.)
The following thresholds verify:
* ⌊2 N / 3⌋ + 1 = ⌊(6f + 5 + 2k) / 3⌋ = 2f + 1 + k + ⌊(2 - k) / 3⌋ = N - f
* ⌊(N + 2) / 3⌋= ⌊(3f + 3 + k) / 3⌋ = f + 1 + ⌊k / 3⌋ = f + 1
```

Reply:
very good.